src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { AddTaskButton } from "../../Atoms/AddTaskButton/index";
+import { Task } from "../../Molecules/Task/index";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+
+export const TodoCard = () => {
+  const [taskList, setTaskList] = useState([]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList([...taskList, { name: "", initializing: true }]);
+  };
+
+  const onTaskComplete = (popTargetIndex) => {
+    setTaskList(taskList.filter((task, index) => index !== popTargetIndex));
+  };
+
+  const onTaskNameChange = (value, editTargetIndex) => {
+    if (value === "") {
+      onTaskComplete(editTargetIndex);
+    } else {
+      setTaskList(() => {
+        const newTaskList = [...(taskList || [])];
```

Comment:
```suggestion
        const newTaskList = [...taskList];
```
`taskList` は 空配列 で初期化しているので、↑の書き方で問題ないと思います。